### DISCUSSION TITLE
Editing a message should populate the message box with the old contents

### DISCUSSION THREAD

**Discussion by hseg** [d99kris/nchat#188] [Ideas]
Often, the edit is just a typo, which, if the message is particularly long, can be annoying to copy/paste/edit

**Comment by d99kris** [comment]
Hi @hseg - this should already be the case as far as I know. One selects a message (up/down keys) then when pressing `ctrl-z` the message input field gets populated with the original contents. One can modify the contents and press `ctrl-x` to save the edit, or press `ctrl-c` to cancel the edit. Are you seeing a different behavior? If yes, are you editing a plain text message (or with images/etc)? Which nchat version are you using?

**Comment by hseg** [comment]
On Thu, Feb 22, 2024 at 04:52:09AM -0800, Kristofer Berggren wrote:
> Hi @hseg - this should already be the case as far as I know. One selects a message (up/down keys) then when pressing `ctrl-z` the message input field gets populated with the original contents. One can modify the contents and press `ctrl-x` to save the edit, or press `ctrl-c` to cancel the edit. Are you seeing a different behavior? If yes, are you editing a plain text message (or with images/etc)? Which nchat version are you using?

Hrm, can't reproduce anymore. Given that I also had contacts not sync up when this occured, I suspect this is more to do with the first time login being a little buggy than anything in particular with nchat.

But yes, I was editing a text message, and the message box wasn't populating at all -- needed to copy/paste the message manually.

**Comment by Disonantemus** [comment]
> Hi @hseg - this should already be the case as far as I know. One selects a message (up/down keys) then when pressing `ctrl-z` the message input field gets populated with the original contents. One can modify the contents and press `ctrl-x` to save the edit, or press `ctrl-c` to cancel the edit. Are you seeing a different behavior? If yes, are you editing a plain text message (or with images/etc)? Which nchat version are you using?

This works 4 me.

**Reply by d99kris** [reply]
Ok 👍
